diff --git a/ParaViewCore/ClientServerCore/Core/CMakeLists.txt b/ParaViewCore/ClientServerCore/Core/CMakeLists.txt
index f98b22e..01f2260 100644
--- a/ParaViewCore/ClientServerCore/Core/CMakeLists.txt
+++ b/ParaViewCore/ClientServerCore/Core/CMakeLists.txt
@@ -106,6 +106,7 @@ if (PARAVIEW_ENABLE_PYTHON)
     vtkPythonExtractSelection.cxx
     vtkPythonProgrammableFilter.cxx
     vtkPythonAnnotationFilter.cxx
+    vtkPythonSelector.cxx
     )
 endif()
