diff --git a/ParaViewCore/ClientServerCore/Core/CMakeLists.txt b/ParaViewCore/ClientServerCore/Core/CMakeLists.txt
index 4f283bd..f05aa2b 100644
--- a/ParaViewCore/ClientServerCore/Core/CMakeLists.txt
+++ b/ParaViewCore/ClientServerCore/Core/CMakeLists.txt
@@ -34,7 +34,6 @@ set (Module_SRCS
   vtkMPIMToNSocketConnection.cxx
   vtkMPIMToNSocketConnectionPortInformation.cxx
   vtkNetworkAccessManager.cxx
-  vtkPConvertSelection.cxx
   vtkProcessModuleAutoMPI.cxx
   vtkProcessModule.cxx
   vtkPVAlgorithmPortsInformation.cxx
@@ -48,8 +47,6 @@ set (Module_SRCS
   vtkPVDataSizeInformation.cxx
   vtkPVDisableStackTraceSignalHandler.cxx
   vtkPVEnableStackTraceSignalHandler.cxx
-  vtkPVExtractSelection.cxx
-  vtkPVExtractArraysOverTime.cxx
   vtkPVFileInformationHelper.cxx
   vtkPVGenericAttributeInformation.cxx
   vtkPVInformation.cxx
diff --git a/ParaViewCore/ClientServerCore/Core/module.cmake b/ParaViewCore/ClientServerCore/Core/module.cmake
index f17c4df..f0db074 100644
--- a/ParaViewCore/ClientServerCore/Core/module.cmake
+++ b/ParaViewCore/ClientServerCore/Core/module.cmake
@@ -18,7 +18,5 @@ vtk_module(vtkPVClientServerCoreCore
     ParaViewCore
   DEPENDS
-    vtkFiltersExtraction
-    vtkFiltersParallel
     # Explicitely list (rather than transiently through
     # vtkPVVTKExtensionsCore) because it allows us to turn of wrapping
     # of vtkPVVTKExtensionsCore off but still satisfy API dependcy.
