diff --git a/ParaViewCore/ServerManager/SMApplication/CMakeLists.txt b/ParaViewCore/ServerManager/SMApplication/CMakeLists.txt
index 8312828..18de793 100644
--- a/ParaViewCore/ServerManager/SMApplication/CMakeLists.txt
+++ b/ParaViewCore/ServerManager/SMApplication/CMakeLists.txt
@@ -100,6 +100,7 @@ target_link_libraries(vtkPVServerManagerApplication
 #------------------------------------------------------------------------------
 SET(requestedResourceFiles
   ${CMAKE_CURRENT_SOURCE_DIR}/Resources/3d_widgets.xml
+  ${CMAKE_CURRENT_SOURCE_DIR}/Resources/catalyst.xml
   ${CMAKE_CURRENT_SOURCE_DIR}/Resources/filters.xml
   ${CMAKE_CURRENT_SOURCE_DIR}/Resources/internal_writers.xml
   ${CMAKE_CURRENT_SOURCE_DIR}/Resources/readers.xml
