diff --git a/VTK/CMakeLists.txt b/VTK/CMakeLists.txt
index d8a5713..d66bdfe 100644
--- a/VTK/CMakeLists.txt
+++ b/VTK/CMakeLists.txt
@@ -268,7 +268,7 @@ endif()

 #-----------------------------------------------------------------------------
 # Add the Remote Subdirectory
-add_subdirectory(Remote)
+# add_subdirectory(Remote)

 #-----------------------------------------------------------------------------
 # Do we try to use system libraries by default?
