diff --git a/VTK/Filters/Extraction/CMakeLists.txt b/Filters/Extraction/CMakeLists.txt
index 359b4e7..65de7c9 100644
--- a/VTK/Filters/Extraction/CMakeLists.txt
+++ b/VTK/Filters/Extraction/CMakeLists.txt
@@ -1,6 +1,5 @@
 set(Module_SRCS
   vtkConvertSelection.cxx
-  vtkExtractArraysOverTime.cxx
   vtkExtractBlock.cxx
   vtkExtractCells.cxx
   vtkExtractDataOverTime.cxx
diff --git a/VTK/Filters/Extraction/module.cmake b/Filters/Extraction/module.cmake
index c578b62..e979d69 100644
--- a/VTK/Filters/Extraction/module.cmake
+++ b/VTK/Filters/Extraction/module.cmake
@@ -4,8 +4,6 @@ vtk_module(vtkFiltersExtraction
   DEPENDS
     vtkCommonExecutionModel
     vtkFiltersCore
-    vtkFiltersGeneral
-    vtkFiltersStatistics
   TEST_DEPENDS
     vtkIOLegacy
     vtkIOXML
