diff --git a/Wrapping/Python/CMakeLists.txt b/Wrapping/Python/CMakeLists.txt
index 969410d..9a0cbdf 100644
--- a/Wrapping/Python/CMakeLists.txt
+++ b/Wrapping/Python/CMakeLists.txt
@@ -71,9 +71,9 @@ file (COPY \"${ParaView_SOURCE_DIR}/VTK/Wrapping/Python/vtk/util/numpy_support.py\"
       USE_SOURCE_PERMISSIONS)
 
 # copy ParaView's default lookup tables
-file(COPY "${CMAKE_SOURCE_DIR}/Qt/Components/Resources/XML/ColorMaps.xml"
-  DESTINATION "${PV_PYTHON_MODULE_BINARY_DIR}"
-  USE_SOURCE_PERMISSIONS)
+#file(COPY "${CMAKE_SOURCE_DIR}/Qt/Components/Resources/XML/ColorMaps.xml"
+#  DESTINATION "${PV_PYTHON_MODULE_BINARY_DIR}"
+#  USE_SOURCE_PERMISSIONS)
 
 # Byte compile the Python files.
 CONFIGURE_FILE(${PV_PYTHON_MODULE_SOURCE_DIR}/compile_all_pv.py.in"
